mappings/net/minecraft/entity/passive/WanderingTraderEntity.mapping

```diff
@@ -1 +1,14 @@
 CLASS ars net/minecraft/entity/passive/WanderingTraderEntity
+	CLASS ars$a WanderToTargetGoal
+		FIELD b proximityDistance D
+		FIELD c speed D
+		METHOD a isNotCloseEnoughTo (Les;D)Z
```

Comment:
`isTooFarFrom`

Reply:
`isOutOfRange`?

Reply:
Ah. When the wandering trader is not in the double distance from the pos, the goal will execute.

Reply:
renamed to isTooFarFrom